index.js

```diff
@@ -20,12 +26,13 @@ if (!parameters.options) {
   return;
 }
 
-const options = JSON.parse(fs.readFileSync(absolutePath(parameters.options)));
+const options = JSON.parse(fs.readFileSync(/* absolutePath( */parameters.options)/* ) */);
+
 if (!options) {
   return;
 }
 
-const metadataFilePath = absolutePath(options["metadataFilePath"]);
+const metadataFilePath = /* absolutePath( */options["metadataFilePath"]/* ) */;
```

Comment:
Should avoid commented implementations (source control is a better to see changes)